envoy_iptables/proxy_init.sh

```diff
@@ -56,54 +56,54 @@ if [[ -z "${ENVOY_IN_PORT-}" ]] || [[ -z "${ENVOY_UID-}" ]]; then
 fi
 
 # Create a new chain for redirecting inbound traffic to Envoy port
-iptables -t nat -N ENVOY_IN_REDIRECT                                                    -m comment --comment "envoy/redirect-inbound-chain"
```

Comment:
🤔 Could we return from the script if this chain already exists?

Reply:
Yes, we could. But nothing can make sure the rule on this chain ready exists.

Reply:
Is it a likely scenario that the script run has been interrupted midway? I didn't think so, but I might be wrong.